package.json

```diff
@@ -73,7 +73,9 @@
     "@babel/preset-env": "^7.20.2",
     "@babel/preset-typescript": "^7.18.6",
     "@rollup/plugin-babel": "^6.0.3",
+    "@rollup/plugin-commonjs": "^24.0.1",
     "@rollup/plugin-json": "^6.0.0",
+    "@rollup/plugin-node-resolve": "^15.0.1",
```

Comment:
do we need those 2?
i m planning to stop rolling up for node.

Reply:
maybe, if we want to use rollup to bundle test files one day, but for now I reverted.
The plugins for rollup and karma are shit